cmd/kube-controller-manager/app/options/options.go

```diff
@@ -202,8 +202,10 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet, allControllers []string, disabled
 	fs.BoolVar(&s.EnableContentionProfiling, "contention-profiling", false, "Enable lock contention profiling, if profiling is enabled")
 	fs.StringVar(&s.ClusterName, "cluster-name", s.ClusterName, "The instance prefix for the cluster")
 	fs.StringVar(&s.ClusterCIDR, "cluster-cidr", s.ClusterCIDR, "CIDR Range for Pods in cluster.")
+	fs.StringSliceVar(&s.ClusterCIDRs, "cluster-cidrs", s.ClusterCIDRs, "list of CIDR Ranges for Pods in cluster.")
 	fs.StringVar(&s.ServiceCIDR, "service-cluster-ip-range", s.ServiceCIDR, "CIDR Range for Services in cluster.")
 	fs.Int32Var(&s.NodeCIDRMaskSize, "node-cidr-mask-size", s.NodeCIDRMaskSize, "Mask size for node cidr in cluster.")
+	fs.IntSliceVar(&s.NodeCIDRMaskSizes, "node-cidr-mask-sizes", s.NodeCIDRMaskSizes, "Mask sizes for node cidrs in cluster.")
```

Comment:
As with the above, we eventually need to allow the per-node CIDR size to vary.  For now, this is OK, but it has to parallel the design above